### ISSUE TITLE
Simplify. Spacing

### ISSUE THREAD

**Issue by kalverra** [gonzalo-github/prj-rev-bwfs-tea-cozy#4]
https://github.com/gonzalo-github/prj-rev-bwfs-tea-cozy/blob/master/Tea_Cozy_Project/index.html#L134-L175

Some unnecessary spacing and comments here, keep it to one line, too much actually hurts readability. Only use spacing and comments to separate large portions of code.

**Comment by gonzalo-github** [comment]
Agree. I acknowledge that many comments may undermine the understanding of the code.
